doc/swagger.v1.yml

```diff
@@ -285,6 +310,51 @@ definitions:
         format: date-time
         description: when the transaction was entered
     required: ['id', 'user_id', 'user_name', 'amount', 'note', 'created_at']
+    
+  FinancialSummary:
+    type: object
+    properties:
+      id:
+        type: integer
+      user_id:
+        type: 'integer'
+        default: null
+        description: id of user who entered the transaction (may be <tt>null</tt> for deleted users or 0 for a system user)
+      user_name:
+        type: 'string'
+        default: null
+        description: name of user who entered the transaction (may be <tt>null</tt> or empty string for deleted users or system users)
+      oder_id:
+        type: 'integer'
+        default: null
+        description: id of order group, user is assigned to
+      credit:
+        type: number
+        default: 0
+        description: actual available credit of user/ordergroup
+      balance:
+        type: 'integer'
+        default: 0
+        description: credit + open claim
+      classes:
+        type: array
+        default: null
+        description: array of transaction classes and their credit, null if no transaction classes are defined
```

Comment:
In my memory there always is one transaction class, but it's good to handle the case when there isn't.
`null` or empty array?

Reply:
As you prefer.

Lets use empty then, to not use a non existing array?

Reply:
As a front-end developer, I would prefer an empty array - less `null` checking.
In a way, the array does exist, because there is a list of financial transaction classes, which is empty.

But I'll leave it to you, as a user of the API, to see what is most logical.